src/@chakra-ui/gatsby-plugin/styles.ts

```diff
@@ -19,15 +19,25 @@ const styles = {
       bg: mode("white", "gray.700")(props),
       lineHeight: "1.6rem",
     },
+    // styles used by markdown pages - we can't use `ListItem` in them.
+    li: {
```

Comment:
Ideally we should not use global styles at all for this. I'm investigating how to avoid it.